Slingshot.Api/Controllers/ARMController.cs

```diff
@@ -339,6 +339,14 @@ private void DeleteResourceGroup(string subscriptionId, string token, string rgN
             returnObj["hasRepoAccess"] = hasRepoAccessTask.Result;
             returnObj["repositoryUrl"] = repo.RepositoryUrl;
             returnObj["repositoryDisplayUrl"] = repo.RepositoryDisplayUrl;
+            returnObj["isManualIntegration"] = true;
+
+            var queryStrings = HttpUtility.ParseQueryString(repositoryUrl);
+            bool isManualIntegration = true;
+            if (bool.TryParse(queryStrings["im"], out isManualIntegration))
```

Comment:
I would call the flag `manual`. It's clearer, and I don't think we need to make the URL as small as possible. Same for the branch, where we currently have `at`. I'd just call it `branch`.